cmd/util.go

```diff
@@ -95,16 +96,16 @@ func (l *listIterator) Next() (string, error) {
 }
 
 // NewReaderIterator returns a new iterator from a reader.
-func NewReaderIterator(rd io.Reader, sep byte) Iterator {
-	return &readerIterator{rd: bufio.NewReader(rd), sep: sep}
+func NewReaderIterator(rd io.Reader, sep byte, cutset string) Iterator {
+	return &readerIterator{rd: bufio.NewReader(rd), sep: sep, cutset: string(sep) + cutset}
 }
 
 func (r *readerIterator) Next() (string, error) {
 	s, err := r.rd.ReadString(r.sep)
 	if err == io.EOF && s != "" {
 		return s, nil
 	}
-	return strings.Trim(s, string(r.sep)), err
+	return strings.Trim(s, r.cutset), err
```

Comment:
A `strings.TrimSpace` would probably have been simpler, but whatever you prefer.